common.js

```diff
@@ -2716,6 +2716,44 @@ const allSoloCars = {
         '2004': ['ep'],
     },
   },
+  'Scion': {
+    'FR-S (incl. Release Series 1.0 and 2.0)': {
+        'all': ['ssc','stx','csp'],
+    },
+    'FR-S *Limited Prep*': {
```

Comment:
You can remove this for now, I haven't figured out a great way to incorporate limited prep into the website yet.